src/commands/setup-emulators-pubsub.ts

```diff
@@ -0,0 +1,10 @@
+import * as Command from "../command";
+
+const downloadEmulator = require("../emulator/download");
+const emulatorName = "pubsub";
```

Comment:
s/emulatorName/EMULATOR_NAME (constant case)

and this declaration should be its own block

Reply:
:+1: sorry about all the style in this file I just copied the `setup-database-emulator` file and didn't think about it.